docs/visualization/visualization_api/callback_scheduling_visualization.md

```diff
@@ -1,6 +1,7 @@
 # Callback Scheduling Visualization
 
-Callback Scheduling Visualization will show you callback scheduling of targets such as a Node, Executor, and Callbackgroup.
+To investigate node behaviors, it is needed to visualize the interaction between callbacks.
```

Comment:
I think callback scheduling visualization doesn't show interaction between callbacks (It's highly depends on implementations).

Reply:
I added a link.
https://github.com/tier4/CARET_doc/pull/98/commits/852b5e04f782cfa82109af7d22f6ead698c6b51a